training/distributed_training/pytorch/model_parallel/gpt-j/train_gptj_smp_notebook.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        "dataset_config_name": "sst2",
</pre><p>Where is the origin of the SST2 dataset? Can you change it so that it pulls from the official example notebook S3 bucket (i.e. sagemaker-example-files)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Origin is Hugging Face Data set - dataset = load_dataset('glue', 'sst2'). Its better to keep this way to keep it according to Hugging Face standards. Customers can change it if they like to point to their own datasets.